src/unix/notbsd/mod.rs

```diff
@@ -253,6 +240,19 @@ s_no_extra_traits!{
         __ss_pad2: [u8; 128 - 2 * 8],
     }
 
+    pub struct sockaddr_alg {
```

Comment:
Why was this code moved, it makes it hard to inspect what changed.

Reply:
Only `SOL_ALG` was moved, other types were not defined prior to this PR

Reply:
Weird, you are right. I'm on the train without quality internet, and github was showing me a negative diff here for some reason. Forget that.